packages/broker/src/ConfigWizard.ts

```diff
@@ -219,65 +185,111 @@ export const selectDestinationPathPrompt = {
             answers.fileExists = existsSync(input)
 
             return true
-        } catch (e) {
+        } catch (e: any) {
             return e.message
         }
     }
 }
 
-const selectValidDestinationPath = async (): Promise<inquirer.Answers> => {
-    const answers = await inquirer.prompt([selectDestinationPathPrompt])
+const selectStoragePath = async (): Promise<inquirer.Answers> => {
+    const answers = await inquirer.prompt([selectStoragePathPrompt])
 
     if (answers.fileExists) {
         const overwriteAnswers = await inquirer.prompt([
             {
                 type: 'confirm',
                 name: 'confirmOverwrite',
-                message: `The selected destination ${answers.selectDestinationPath} already exists, do you want to overwrite it?`,
+                message: `The selected destination ${answers.selectStoragePath} already exists, do you want to overwrite it?`,
                 default: false,
             }
         ])
 
         if (!overwriteAnswers.confirmOverwrite) {
-            return selectValidDestinationPath()
+            return selectStoragePath()
         }
     }
 
     return answers
 }
 
+export const getPrivateKey = (answers: inquirer.Answers): string => {
+    return (answers.generateOrImportPrivateKey === PRIVATE_KEY_SOURCE_IMPORT) ? answers.importPrivateKey : Wallet.createRandom().privateKey
+}
+
+export const getConfig = (privateKey: string, pluginsAnswers: inquirer.Answers): any => {
+    const config = { ... CONFIG_TEMPLATE, plugins: { ... CONFIG_TEMPLATE.plugins } }
+    config.ethereumPrivateKey = privateKey
+
+    const pluginNames = Object.keys(PLUGIN_NAMES)
```

Comment:
The keys are not plugin names, but some IDs (compare to line `225` where we get the plugin name by that id)